### PR TITLE
BrenKt-Creation-DMARCguide-FR

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -954,6 +954,7 @@
                 + [Customising the DNS servers for an OVHcloud domain name](web/domains/glue_registry)
                 + [Adding an SPF record to the configuration of your domain name](web/domains/dns_zone_spf)
                 + [Adding a DKIM record to the configuration of your domain name](web/domains/dns_zone_dkim)
+                + [Adding a DMARC record to the configuration of your domain name](web/domains/dns_zone_dkim)
```
Comment (OVH-Team-Guides) [inline pages/index.md]:
```suggestion
                + [Adding a DMARC record to the configuration of your domain name](web/domains/dns_zone_dmarc)
```

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+- **Règle pour les sous-domaines (sp=)** : politique à adopter par le destinataire pour tous les sous-domaines. Elle s'applique uniquement aux sous-domaines du domaine interrogé et non au domaine lui-même. Sa syntaxe est identique à celle de la balise « p » définie ci-dessus. Si cette balise est absente, la politique spécifiée par la balise « p » est appliquée pour les sous-domaines.
+
+- **Mode d'alignement pour SPF (aspf=)** (la valeur par défaut est `r`) : indique le mode d'allignement SPF. Les valeurs sont les suivantes :
+    - `r`(relaxed) pour le mode souple : les e-mails échouant à l'authentification SPF sont marqués comme « indésirables » par le serveur destinataire.
+    - `s`(strict) pour le mode strict : les e-mails échouant à l'authentification SPF sont rejetés par le serveur destinataire.
+
+> [!primary]
+>
+> Dans le cadre des mécanismes d'authentification SPF et DKIM, l'**alignement** fait référence à la correspondance entre le nom de domaine (et/ou la signature du domaine) utilisé lors de l'envoi, avec le nom de domaine inscrit dans ces mécanismes.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zone_dmarc/guide.fr-fr.md]:
```suggestion
> Dans le cadre des mécanismes d'authentification SPF et DKIM, l'**alignement** fait référence à la correspondance entre le nom de domaine (et/ou la signature du domaine) utilisé lors de l'envoi **et** le nom de domaine inscrit dans ces mécanismes.
```

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+>
+> **exemple**
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zone_dmarc/guide.fr-fr.md]:
```suggestion
> **Exemple :** 
```

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    - `r`(relaxed) pour le mode souple : les e-mails échouant à l'authentification SPF sont marqués comme « indésirables » par le serveur destinataire.
+    - `s`(strict) pour le mode strict : les e-mails échouant à l'authentification SPF sont rejetés par le serveur destinataire.
+
+> [!primary]
+>
+> Dans le cadre des mécanismes d'authentification SPF et DKIM, l'**alignement** fait référence à la correspondance entre le nom de domaine (et/ou la signature du domaine) utilisé lors de l'envoi, avec le nom de domaine inscrit dans ces mécanismes.
+>
+> **exemple** 
+>
+> - **Aligné**: lorsque l'adresse *john.smith@mydomain.ovh* transmet un message depuis le service e-mail attaché au nom de domaine *mydomain.ovh* et que les mécanismes d'authentification SPF et DKIM ont été configurés, nous obtiendrons un résultat aligné.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zone_dmarc/guide.fr-fr.md]:
```suggestion
> - **Aligné** : lorsque l'adresse *john.smith@mydomain.ovh* transmet un message depuis le service e-mail attaché au nom de domaine *mydomain.ovh* et que les mécanismes d'authentification SPF et DKIM ont été configurés, on obtient un résultat aligné.
```

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+> - **Partiellement aligné**: lorsque l'adresse *john.smith@subdomain.mydomain.ovh* transmet un message depuis le service e-mail attaché au nom de domaine *mydomain.ovh*, mais que les mécanismes d'authentification SPF et DKIM ont été configurés uniquement sur le domaine principal, à savoir *mydomain.ovh*, nous obitiendrons un résultat partiellement aligné.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zone_dmarc/guide.fr-fr.md]:
```suggestion
> - **Partiellement aligné** : lorsque l'adresse *john.smith@subdomain.mydomain.ovh* transmet un message depuis le service e-mail attaché au nom de domaine *mydomain.ovh*, mais que les mécanismes d'authentification SPF et DKIM ont été configurés uniquement sur le domaine principal (c'est à dire *mydomain.ovh*), on obtient un résultat partiellement aligné.
```

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+> - **Echec des mécanismes d'authentification**: l'expéditeur tente d'envoyer un e-mail en tant que *john.smith@mydomain.ovh* en passant par une autre adresse, tel que *robert@example.com* ou encore en utilisant un service d'envoie d'e-mail qui n'est pas listé dans le SPF. Alors les mécanismes d'authentification SPF et DKIM renveront un échec comme résultat.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zone_dmarc/guide.fr-fr.md]:
```suggestion
> - **Echec des mécanismes d'authentification** : l'expéditeur tente d'envoyer un e-mail en tant que *john.smith@mydomain.ovh* en passant par une autre adresse (telle que *robert@example.com*) ou en utilisant un service d'envoi d'e-mail qui n'est pas listé dans le SPF. Dans ce cas, les mécanismes d'authentification SPF et DKIM renvoient un échec comme résultat.
```

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
+---
+title: Configurer un enregistrement DMARC sur votre nom de domaine
```
Comment (Kazarus4) [inline pages/web/domains/dns_zone_dmarc/guide.fr-fr.md:2]:
```suggestion
title: "Configurer un enregistrement DMARC sur votre nom de domaine"
```

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+excerpt: Découvrez comment fonctionne DMARC et comment le mettre en place pour votre service e-mail
```
Comment (Kazarus4) [inline pages/web/domains/dns_zone_dmarc/guide.fr-fr.md:3]:
```suggestion
excerpt: "Découvrez comment fonctionne DMARC et comment le mettre en place pour votre service e-mail"
```

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+   color: #000;
+ }
+ .small {
+     font-size: 0.90em;
+ }
+</style>
+
+## Objectif
+
+L'enregistrement DMARC (**D**omain-based **M**essage **A**uthentication, **R**eporting, and **C**onformance) est un mécanisme de sécurité e-mail. Il s'appuie sur le résultat des vérifications [SPF](/pages/web/domains/dns_zone_spf) et [DKIM](/pages/web/domains/dns_zone_dkim).
```
Comment (Kazarus4) [inline pages/web/domains/dns_zone_dmarc/guide.fr-fr.md]:
```suggestion
L'enregistrement **D**omain-based **M**essage **A**uthentication, **R**eporting and **C**onformance (**DMARC **), est un mécanisme de sécurité e-mail. Il s'appuie sur le résultat des vérifications [SPF](/pages/web/domains/dns_zone_spf) et [DKIM](/pages/web/domains/dns_zone_dkim).
```

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md [hunk 10]
```diff
@@ -0,0 +1,180 @@
...
+
+- Disposer d'un accès à la gestion du nom de domaine (attaché à votre solution e-mail) depuis l'[espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr).
+- Disposer d'une solution e-mail compatible avec le DMARC.
+- Un des mécanismes d'authentification, [SPF](/pages/web/domains/dns_zone_spf) et/ou [DKIM](/pages/web/domains/dns_zone_dkim) doit être configuré dans la zone DNS du nom de domaine de l'offre e-mail.
+
+## En pratique
+
+L' enregistrement DMARC contient des informations sur la politique à appliquer pour les e-mails malveillants qui tentent d'usurper votre nom de domaine.<br>
+DMARC interroge les mécanismes d'authentification [SPF](/pages/web/domains/dns_zone_spf) et [DKIM](/pages/web/domains/dns_zone_dkim) pour vérifier les e-mails entrants.<br>
+Le résultat de ces vérifications SPF et/ou DKIM est traduit par DMARC en mesures à prendre lorsqu'un e-mail échoue aux contrôles, comme la mise en quarantaine ou le rejet des e-mails concernés.
```
Comment (Kazarus4) [inline pages/web/domains/dns_zone_dmarc/guide.fr-fr.md]:
```suggestion
Le résultat de ces vérifications SPF et/ou DKIM est traduit par DMARC en *mesures à prendre* lorsqu'un e-mail échoue aux contrôles, comme la mise en quarantaine ou le rejet des e-mails concernés.
```

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md [hunk 11]
```diff
@@ -0,0 +1,180 @@
...
+
+Après lecture des instructions de l'enregistrement DMARC du nom de domaine **mydomain.ovh** par le serveur de réception de **« otherdomain.ovh »**, les e-mails seront soit délivrés vers l'adresse **recipient@otherdomain.ovh**, soit marqués comme « SPAM » ou rejetés.
+
+![dmarc](images/dns-dmarc-diagram.png){.thumbnail}
+
+### Configurer le DMARC 
+
+Vous devez d'abord activer le DMARC sur le service e-mail associé à votre nom de domaine.
+
+Pour configurer la zone DNS du nom de domaine de votre service e-mail, rendez-vous dans l'[espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr){.external}, dans la section `Noms de domaine`{.action} à gauche.
```
Comment (Kazarus4) [inline pages/web/domains/dns_zone_dmarc/guide.fr-fr.md]:
```suggestion
Pour configurer la zone DNS du nom de domaine de votre service e-mail, connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr){.external} puis rendez-vous dans la partie `Web Cloud`{.action}. Dans la colonne de gauche, sélectionnez, dans la section `Noms de domaine`{.action}, le nom de domaine concerné puis cliquez sur l'onglet `Zone DNS`{.action} pour accéder à votre zone DNS.
```

**Review by Kazarus4 — COMMENTED** [review]
review done from the begining to line 70. I'll be continue tomorrow the rest of the guide.

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md (hunk 10 above)
Comment (OVH-Team-Guides) [reply pages/web/domains/dns_zone_dmarc/guide.fr-fr.md]:
```suggestion
Le résultat de ces vérifications SPF et/ou DKIM est traduit par DMARC en « mesures à prendre » lorsqu'un e-mail échoue aux contrôles, comme la mise en quarantaine ou le rejet des e-mails concernés.
```

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md [hunk 12]
```diff
@@ -0,0 +1,180 @@
...
+
+Il y a deux façons de configurer le DMARC dans votre zone DNS OVHcloud :
+
+- Par le biais d'un [enregistrement DMARC](#dmarc-record). Cet enregistrement permet une configuration simplifiée du DMARC, vous n'aurez qu'à compléter les champs avec les paramètres DMARC nécessaires à votre configuration. Cet enregistrement est lu comme un enregistrement TXT par les serveurs DNS.
+
+- Par le biais d'un [enregistrement TXT](#txt-record). Cet enregistrement standard peut être utilisé dans le cadre de la configuration du DMARC depuis l'espace client OVHcloud, il vous permettra d'intégrer l'ensemble des balises de paramétrage DMARC y compris celles absentes via l'enregistrement DMARC OVHcloud. Il nécessite toutefois de bien respecter les règles de syntaxe du protocole DMARC.
+
+#### Enregistrement DMARC <a name="dmarc-record"></a>
+
+Vous pouvez ajouter l'enregistrement DMARC à votre zone DNS depuis l'[espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr){.external}. Cliquez sur `Noms de domaine`{.action} puis choisissez le nom de domaine concerné. Positionnez-vous enfin sur l'onglet `Zone DNS`{.action}.
```
Comment (Kazarus4) [inline pages/web/domains/dns_zone_dmarc/guide.fr-fr.md]:
```suggestion
Vous pouvez ajouter l'enregistrement DMARC à votre zone DNS depuis l'espace client OVHcloud. Pour cela, , connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr){.external} puis rendez-vous dans la partie `Web Cloud`{.action}. Dans la colonne de gauche, sélectionnez, dans la section `Noms de domaine`{.action}, le nom de domaine concerné puis cliquez sur l'onglet `Zone DNS`{.action} pour accéder à votre zone DNS..
```

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md (hunk 3 above)
Comment (Kazarus4) [inline pages/web/domains/dns_zone_dmarc/guide.fr-fr.md]:
```suggestion
> **exemples** 
```

File: pages/web/domains/dns_zone_dmarc/guide.fr-fr.md [hunk 13]
```diff
@@ -0,0 +1,180 @@
...
+
+- **aspf=s** : le mode d'alignement d'identifiant SPF requis est « strict ». Cela signifie que l'identifiant SPF du domaine aligné doit correspondre exactement à l'adresse IP émettrice du message.
+
+- **adkim=r** : le mode d'alignement d'identifiant DKIM requis par le propriétaire du domaine est « relaxed » (mode souple). Dans ce mode, DKIM doit fournir une signature valide et l'identifiant de l'en-tête « From » peut être partiellement aligné.
+
+- **ri=86400** : définit l'intervalle demandé entre les rapports agrégés, en secondes. Dans ce cas, un rapport agrégé doit être généré au moins une fois toutes les 86400 secondes (soit une fois par jour).
+
+## Aller plus loin
+
+Échangez avec notre communauté d'utilisateurs sur [https://community.ovh.com](https://community.ovh.com).
```
Comment (Kazarus4) [inline pages/web/domains/dns_zone_dmarc/guide.fr-fr.md]:
```suggestion
Pour des prestations spécialisées (référencement, développement, etc), contactez les [partenaires OVHcloud](https://partner.ovhcloud.com/fr/directory/).

Si vous souhaitez bénéficier d'une assistance à l'usage et à la configuration de vos solutions OVHcloud, nous vous proposons de consulter nos différentes [offres de support](https://www.ovhcloud.com/fr/support-levels/).

Échangez avec notre communauté d'utilisateurs sur <https://community.ovh.com>.
```

**Review by Kazarus4 — COMMENTED** [review]
peer rv done for me

**Pull request merged**